README.md

```diff
@@ -26,54 +26,56 @@ $ mv kperf /usr/local/bin/
+
+```shell script
+# Create a namespace for kperf to create Services in it
+kubectl create ns {namespace-name}
+
+# Create namespaces from test-1 to test-10 for kperf to create Services distributed in them
+for name in {1..10};do kubectl create ns test-$name;done
 ```
+
+### generate Knative Service deployment load
+```shell script
 # Generate total 30 knative service, for each 15 seconds create 10 ksvc with 5 concurrency in namespace test1, test2 and test3, and the ksvc names are ktest1, ktest2.....ktest29.
 $ kperf service generate -n 30 -b 10 -c 5 -i 15 --nsPrefix test --nsRange 1,3 --svcPrefix ktest --maxScale 1 --minScale 1
 
-Creating ksvc ktestsss0 in namespace test1
-Creating ksvc ktestsss1 in namespace test2
+Creating ksvc ktest-0 in namespace test-1
+Creating ksvc ktest-1 in namespace test-2
 ...
 ...
-Creating ksvc ktestsss29 in namespace test3
-```
+Creating ksvc ktest-29 in namespace test-3
 ```
+
+```shell script
 # Generate total 30 knative service, for each 15 seconds create 10 ksvc with 1 concurrency in namespace test1, test2 and test3, and the ksvc names are ktest1, ktest2.....ktest29. The generation will wait the previous genreated service to be ready for most 10 seconds.
 $ kperf service generate -n 30 -b 10 -c 5 -i 15 --nsPrefix test --nsRange 1,3 --svcPrefix ktest --wait --timeout 10s --maxScale 1 --minScale 1
 
-Creating ksvc ktestsss0 in namespace test1
-Creating ksvc ktestsss1 in namespace test2
-...
-...
-Creating ksvc ktestsss29 in namespace test3
-```
-
-### As a Knative developer, I want to clean Knative Service generated for test
-```
-# Delete all ksvc with name prefix ktest in namespaces with name prefix test and index 1,2,3
-$ kperf service clean --nsPrefix test --nsRange 1,3 --svcPrefix ktest
-
-Delete ksvc ktestsss0 in namespace test1
-Delete ksvc ktestsss12 in namespace test1
+Creating ksvc ktests-0 in namespace test-1
+Creating ksvc ktests-1 in namespace tes-2
 ...
-Delete ksvc ktestsss1 in namespace test2
-Delete ksvc ktestsss10 in namespace test2
 ...
-Delete ksvc ktestsss5 in namespace test3
-Delete ksvc ktestsss8 in namespace test3
+Creating ksvc ktests-29 in namespace test-3
 ```
 
-### As a Knative developer, I want to measure Knative Service creation time duration
+### Measure Knative Service deployment time
 - Service Configurations Duration Measurement: time duration for Knative Configurations to be ready
 - Service Routes Duration Measurement: time duration for Knative Routes to be ready
 - Overall Service Ready Measurement: time duration for Knative Service to be ready
+
+Here is a figure of different resources generated for a Knative Service(assuming Istio is the network solution).
 ![resources created by Knative Service](docs/service_creation.png)
 
 **Example 1 Measure Services (for eg. range 1,500)for load test under a specific namespace**
 
-```cassandraql
+```shell script
 $ kperf service measure -n ktest-1 --prefix ktest --range 101,110 --verbose
```

Comment:
According to the latest code:
`prefix ` => `svc-prefix`

Reply:
fixed!